src/cc/torchdistx/deferred_init.cc

```diff
@@ -240,7 +240,8 @@ void Op::validateStack(const Stack& s) const {
                 value.isNone() ||
                 value.isString() ||
                 value.isTensor() ||
-                value.isTuple(),
+                value.isTuple() ||
+                value.isSymInt(),
```

Comment:
I assume SymInt is an immutable type as the comment mentions above?

Reply:
yeah, @Krovatkin confirmed